fdbserver/OldTLogServer_6_0.actor.cpp

```diff
@@ -38,7 +38,11 @@
 #include "fdbserver/LogSystem.h"
 #include "fdbserver/WaitFailure.h"
 #include "fdbserver/RecoveryState.h"
+#include "fdbserver/FDBExecArgs.h"
```

Comment:
Just as a general comment: I don't think we need to support 6.0 TLogs. It would certainly make testing simpler if we only have this feature from 6.2 onwards

Reply:
Yes, I did not want to mess with the simulator test and I have paid the cost of implementing in two places already. So, leaving it as is. If I end up doing lot more development, then I will yank the support for 6.0 logs.